spi_nand_flash/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "0.4.1"
+version: "0.4.2"
```

Comment:
Since your change is `feat(...)`, you need to increase minor version: https://semver.org/